index.html

```diff
@@ -4,9 +4,27 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css"/>
     <title>Weather Report</title>
-</head>
+</head> 
+<h1>Weather Report</h1>
 <body>
-    
+    <div class="location">
+        <h1 class="location">City, State</h1>
+        <p>ICON</p>
+    </div>
+    <div class="temperature">
+        <div class="degree-section">
+        <h2 class ="temperature-degree">Temperature in Farenheit</h2>
+        <div id="floor-container">🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷</div>
+        <button type="button" id="up-button">Up</button>
+        <button type="button" id="down-button">Down</button>
+        <span type="number" id="temperature" value="75" class="orange">70</span>
```

Comment:
The `type` and `value` attributes here only apply to an `input` tag. `span` doesn't know what to do with them.